vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/GridMultiSelectionModel.java

```diff
@@ -115,6 +124,9 @@ void setSelectAllCheckboxVisibility(
      * Returns whether the select all checkbox will be visible with the current
      * setting of
      * {@link #setSelectAllCheckboxVisibility(SelectAllCheckboxVisibility)}.
```

Comment:
settings of ... and the type of data set to the grid.

Reply:
Done